src/app/app.component.ts

```diff
@@ -9,7 +9,7 @@ import { scan, tap } from 'rxjs/operators';
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss'],
-  encapsulation: ViewEncapsulation.ShadowDom
+  encapsulation: ViewEncapsulation.None
```

Comment:
As discussed in PD if the other styles move to styles.scss this can be unchanged

Reply:
addressed in #520